packages/regl-worldview/src/commands/GLText.js

```diff
@@ -354,7 +363,19 @@ function makeTextCommand(alphabet?: string[]) {
         let y = 0;
         let markerInstances = 0;
 
-        const fgColor = marker.colors?.[0] || marker.color || BG_COLOR_LIGHT;
+        const hitmapColor = (markerColor) => {
+          return {
+            r: markerColor?.[0] ?? 0,
```

Comment:
For hitmaps, is it every possible that `markerColor` is not set? That shouldn't happen right? Should we just throw an error in that case instead of defaulting to black? (Or maybe I just don't understand how our hitmaps work?)

Reply:
@janpaul123 right, it shouldn't happen. Here's my rationale for this fix:

The function that calculates the color for hitmap always returns a valid instance (although it's a Vec4[] instead of Color). I might be wrong, but in my tests the color is never black, since that means nothing is selected. So, in the event that we don't get a valid color, we can assume that the object is not selectable, right? If so, using black should be ok.